src/demo/article.html

```diff
@@ -6553,9 +6553,91 @@ <h2>
         References
       </h2>
 
-      <ol>
+      <ol itemtype="https://stencila.github.io/schema/CiteGroup">
         <li>
-          1
+          <ul itemtype="https://stencila.github.io/schema/Cite">
+            <div itemtype="https://schema.org/CreativeWork">
```

Comment:
I believe these `div`s should be wrapped in `li`s first to be semantically correct.\

> Content model:
> Zero or more li and script-supporting elements.
> [HTML 5 Spec](https://www.w3.org/TR/html50/grouping-content.html#the-ul-element)